README.rst

```diff
@@ -753,6 +753,48 @@ Delete device
 
     DELETE /api/v1/controller/device/{id}/
 
+List credentials
+^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    GET api/v1/connection/credential/
```

Comment:
A couple of issues:

- initial slash is missing
- if you guys remember, when we worked on the PR for the `config` sub app, I told that we should use only one prefix for openwisp-controller and not one per subapp, so let's use only `controller` instead of `connection`, eg:
`/api/v1/controller/credential/`.

Applies to all other occurrences.

Reply:
Hi, All these requested chnages have been resolved. :+1: